deephaven/web-client-ui
# fix: Change display of rollup key columns from null to empty string (#1563)

Cherry-pick of #1563 

* Previously, key columns containing non-numeric types would display `null` is the rolled up rows have different values for the same column
    * This behaviour is updated to show an empty cell instead of `null`
* Resolves #1483 

#### Testing Instructions:
1. Run: 
```python 
from deephaven import empty_table, time_table
  
  size = 100
  scale = 1000
  
  random_values_with_null_NaN = empty_table(size).update([
      "MyString=(i%11==0? null : `a`+(int)(scale*(Math.random()*2-1)))"
  ])
```
2. Add the string and char columns and any other arbitrary columns as rollup columns in any order
3. The key column should only show `null` if it actually represents a null value in the column

---------
